tests/advanced_visibility.go

```diff
@@ -2022,9 +2022,8 @@ func (s *AdvancedVisibilitySuite) Test_BuildIdIndexedOnCompletion_UnversionedWor
 
 func (s *AdvancedVisibilitySuite) Test_BuildIdIndexedOnCompletion_VersionedWorker() {
 	// Use only one partition to avoid having to wait for user data propagation later
-	dc := s.testCluster.host.dcClient
-	dc.OverrideValue(s.T(), dynamicconfig.MatchingNumTaskqueueReadPartitions, 1)
-	dc.OverrideValue(s.T(), dynamicconfig.MatchingNumTaskqueueWritePartitions, 1)
+	s.testCluster.host.OverrideDCValue(s.T(), dynamicconfig.MatchingNumTaskqueueReadPartitions, 1)
```

Comment:
I want to simplify this more so you don't need the call to T() and the long path, but that can be in another PR

Reply:
Do you propose adding OverrideDCValue() to FunctionalTestBase?

Reply:
I have made that change as well. Let me know if that is what you were thinking about.

Reply:
yup, exactly what I was thinking, except you can move the s.T() also

Reply:
Done